spec/unit/typesafe/config/conf_parser_spec.rb

```diff
@@ -781,20 +781,26 @@ def assert_comments_at_path_index(comments, conf, path, index)
   # Skipping 'includeURLHeuristically' because we don't support URLs
   # Skipping 'includeURLBasenameHeuristically' because we don't support URLs
 
-  it "acceptBOMStartingFile" do
-    skip("BOM not parsing properly yet; not fixing this now because it most likely only affects windows") do
-      # BOM at start of file should be ignored
-      conf = Hocon::ConfigFactory.parse_file(TestUtils.resource_file("bom.conf"))
-      expect(conf.get_string("foo")).to eq("bar")
-    end
+  it "doesnotacceptUTF8FileNames" do
+    expect { Hocon::ConfigFactory.parse_file(TestUtils.resource_file("ᚠᛇᚻ.conf")) }.to raise_error
   end
```

Comment:
@DavidS `pending` changes semantics between Rspec 2 and 3 (and this repos is currently on the 2 series) ... I've switched things over to `skip` like the others for consistency:

FYI for `pending`
```
The semantics of `RSpec::Core::Pending#pending` are changing in
RSpec 3.  In RSpec 2.x, it caused the example to be skipped. In
RSpec 3, the rest of the example will still be run but is expected
to fail, and will be marked as a failure (rather than as pending)
if the example passes.

Any passed block will no longer be executed. This feature is being
removed since it was semantically inconsistent, and the behaviour it
offered is being made available with the other ways of marking an
example pending.

To keep the same skip semantics, change `pending` to `skip`.
Otherwise, if you want the new RSpec 3 behavior, you can safely
ignore this warning and continue to upgrade to RSpec 3 without
addressing it.
```